ingestion/src/metadata/data_quality/validations/table/sqlalchemy/tableDiff.py

```diff
@@ -75,6 +76,18 @@ def masked(s: str, mask: bool = True) -> str:
     return "***" if mask else s
 
 
+def is_numeric(t: type) -> bool:
+    """Check if a type is numeric.
+
+    Args:
+        t: type to check
+
+    Returns:
+        True if the type is numeric otherwise False
+    """
+    return t in [int, float, Decimal]
+
+
```

Comment:
Are we able to use https://github.com/open-metadata/OpenMetadata/blob/aa7b95a515a6a72b7bad5f49aa19963b85c25be7/ingestion/src/metadata/profiler/orm/registry.py#L159 ?

Reply:
no. that is specific for sqlalchemy types